examples/bri-3/src/bri/zkp/api/dtos/request/createProof.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { IsNotEmpty } from 'class-validator';
+
+// TODO: Revisit validations rules once we move into business logic implementation
+export class CreateProofDto {
+  @IsNotEmpty()
+  id: string;
+
+  @IsNotEmpty()
+  ownerAccountId: string;
+
+  @IsNotEmpty()
+  document: any;
```

Comment:
im ok with `any` but maybe `Object` would be better, your call
we can probably add proper interface sometimes in future too

Reply:
i would use object. in fact I would use `documentObject` where this is made up of `documentObjectType` as `string`, and `documentObjectInput` as ``object` which must conform to the schema defined for the `documentObjectType` ... thoughts? @skosito

Reply:
gives u a lot of flexibility to define custom schemas and use them for validation of the actual object.

Reply:
+1

Reply:
@Therecanbeonlyone1969 yes agree, maybe not essential for now, but we def should not leave it as any